pkgs/applications/misc/moz-phab/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonApplication
+, fetchPypi
+, mercurial
+# build inputs
+, distro
+, glean-sdk
+, python-hglib
+, sentry-sdk
+, setuptools
+}:
+
+buildPythonApplication rec {
+  pname = "MozPhab";
```

Comment:
In python pname, attr and directory name should all match the normalized name.

Reply:
That's a little unfortunate. Is this a convention in NixOS packaging, or somewhere more general? Clearly, it's not a technical limitation, since the current PR works without warnings or errors.
Unfortunate, because the binary is `moz-phab`, and it would make sense to enable it via `moz-phab` package (as opposed to MozPhab).

Reply:
I ended up changing the `pname` in `fetchPypi` only, leaving `moz-phab` consistently everywhere else. I found other crates (adafruit stuff, for example) doing this, so hopefully it's idiomatic.

Reply:
> That's a little unfortunate. Is this a convention in NixOS packaging, or somewhere more general?

Thats for pythonPackages but it won't hurt to do for other packages.

> Clearly, it's not a technical limitation, since the current PR works without warnings or errors.

No it isn't one but not every code smell throws warnings left and right and hard errors usually  only happen when something is severely broken.

Reply:
Is the current approach acceptable? We are using `moz-phab` consistently now in the PR